src/pages/StockPage.js

```diff
@@ -63,6 +63,15 @@ export class StockPage extends React.Component {
     });
   }
 
+  renderCell = (key, item) => {
+    switch (key) {
+      default:
+        return item[key];
+      case 'totalQuantity':
+        return item[key] || 0;
```

Comment:
This will fix it but why doesn't this bug affect other pages such as `CustomerInvoicePage`?

Reply:
great point, i went to investigate, it's because of this: https://github.com/sussol/react-native-generic-table-page/blob/4803b8295357be98a627ea8f0ccab4c14ab4542f/index.js#L237
when no renderCell method is found, this is displayed: `record[key] || ''`, if record is 0 then display will be ''.

I've changed to just use `renderCell = (key, item) => item[key];`